docs/application/dotnet/guides/nui/overview.md

```diff
@@ -20,6 +20,22 @@ NUI offers the following main features:
 -   Easy-to-use [animation framework](animation.md) which hides the complexity of the underlying three-dimensional math
 -   Keyboard, touch, and mouse handling
 
+Some reference applications:
+<table style="text-align:center;">
```

Comment:
As @mijin85cho mentioned, we usually follow markdown tags, kindly let us know if there is any specific reason why you have used HTML tags?

You can refer this file for markdown tags for table.

https://raw.githubusercontent.com/Samsung/tizen-docs/master/docs/application/native/tutorials/details/app-filtering.md